.circleci/config.yml

```diff
@@ -678,11 +678,17 @@ jobs:
       - run:
           name: Check if target branch is op-es
           command: |
+            # Not triggered by a PR
+            if [ -z "$CIRCLE_PULL_REQUEST" ]; then
```

Comment:
`CIRCLE_PULL_REQUEST` looks like a CircleCI related variable, so it's always empty when running on self-hosted machines. Besides, this feature is used by upstream to lock modifications to some contracts when in audit, which we don't actually need, maybe we can directly quit the job but signal a success at the beginning to minimize the diff.

Reply:
Please confirm op-es does not need frozen check @qizhou @qzhodl. We can change runner back to Docker if necessary.

Reply:
Yes, I think we can disable it for new

Reply:
So reverted the job definition to minimise the diff from the upstream, and disabled the reference.